Description: Escape hyphens in audisp-prelude(8)
Author: Russell Coker <russell@coker.com.au>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- audit-2.1.3.orig/audisp/plugins/prelude/audisp-prelude.8
+++ audit-2.1.3/audisp/plugins/prelude/audisp-prelude.8
@@ -1,20 +1,20 @@
 .TH AUDISP-PRELUDE: "8" "Dec 2008" "Red Hat" "System Administration Utilities"
 .SH NAME
-audisp-prelude \- plugin for IDMEF alerts 
+audisp\-prelude \- plugin for IDMEF alerts 
 .SH SYNOPSIS
-.B audisp-prelude [ --test ]
+.B audisp\-prelude [ \-\-test ]
 .SH DESCRIPTION
-\fBaudisp-prelude\fP is a plugin for the audit event dispatcher daemon, audispd, that uses libprelude to send IDMEF alerts for possible Intrusion Detection events. This plugin requires connecting to a prelude-manager to record the events it sends. This plugin will analyze audit events in realtime and send detected events to the prelude-manager for correlation, recording, and display. 
+\fBaudisp\-prelude\fP is a plugin for the audit event dispatcher daemon, audispd, that uses libprelude to send IDMEF alerts for possible Intrusion Detection events. This plugin requires connecting to a prelude\-manager to record the events it sends. This plugin will analyze audit events in realtime and send detected events to the prelude\-manager for correlation, recording, and display. 
 
 Events that are currently supported are: Logins, Forbidden Login Location, Max Concurrent Sessions, Max Login Failures, Forbidden Login Time, SE Linux AVCs, SE Linux Enforcement Changes, Abnormal Program Termination, Promiscuous Socket Changes, and watched account logins.
 
 .SH OPTIONS
 .TP
 .B \-\-test
-Take input from stdin and write prelude events to stdout but does not send them to the prelude-manager. This can be used for debugging or testing the system with suspicious log files when you do not want it to alert or react.
+Take input from stdin and write prelude events to stdout but does not send them to the prelude\-manager. This can be used for debugging or testing the system with suspicious log files when you do not want it to alert or react.
 
 .SH INSTALLATION
-This sensor has to be registered with the prelude-manager before it will work properly. If the prelude-manager is on the same host as the sensor, you will need to open two windows to register. If not, you will have to adjust this example to fit your environment.
+This sensor has to be registered with the prelude\-manager before it will work properly. If the prelude\-manager is on the same host as the sensor, you will need to open two windows to register. If not, you will have to adjust this example to fit your environment.
 
 In one window, type:
 
@@ -24,7 +24,7 @@ In another, type:
 
 .B prelude\-admin registration\-server prelude\-manager
 
-Follow the on-screen instructions to complete the registration.
+Follow the on\-screen instructions to complete the registration.
 
 .SH TIPS
 If you are aggregating multiple machines, you should enable node information in the audit event stream. You can do this in one of two places. If you want computer node names written to disk as well as sent in the realtime event stream, edit the name_format option in /etc/audit/auditd.conf. If you only want the node names in the realtime event stream, then edit the name_format option in /etc/audisp/audispd.conf. Do not enable both as it will put 2 node fields in the event stream.
@@ -61,12 +61,12 @@ To alert on users making exe's in their
 .B \-a exit,always \-S fchmod,chmod \-F dir=/home \-F a1&0111 \-F filetype=file \-k ids\-mkexe\-hi
 
 .SH FILES
-/etc/audisp/plugins.d/au-prelude.conf, /etc/audit/auditd.conf, /etc/audisp/audispd.conf, /etc/audisp/audisp-prelude.conf
+/etc/audisp/plugins.d/au\-prelude.conf, /etc/audit/auditd.conf, /etc/audisp/audispd.conf, /etc/audisp/audisp\-prelude.conf
 .SH "SEE ALSO"
 .BR audispd (8),
-.BR prelude-manager (1),
+.BR prelude\-manager (1),
 .BR auditd.conf (8),
 .BR audispd.conf (8),
-.BR audisp-prelude.conf (5).
+.BR audisp\-prelude.conf (5).
 .SH AUTHOR
 Steve Grubb
